ui/package.json

```diff
@@ -159,7 +159,6 @@
     "qunit": "^2.17.2",
     "qunit-dom": "^1.6.0",
     "route-recognizer": "^0.3.4",
-    "sass": "^1.29.0",
```

Comment:
we can remove sass here, because ember-cli-sass takes care of the sass import. Whatever you define here overrides whatever ember-cli-sass is using and it complicates the setup.